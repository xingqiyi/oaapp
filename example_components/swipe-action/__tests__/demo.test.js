import { webDemoTest } from '../../../tests/shared/demoTest';

webDemoTest('swipe-action');
