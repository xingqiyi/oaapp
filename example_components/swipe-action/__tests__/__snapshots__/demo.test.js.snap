// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders ./components/swipe-action/demo/basic.md correctly 1`] = `
<div
  class="am-list"
>
  <div
    class="am-list-body"
  >
    <div
      style="background-color:gray;"
    >
      <div
        class="am-swipe"
      >
        <div
          class="am-swipe-cover"
        />
        <div
          class="am-swipe-actions am-swipe-actions-left"
        >
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#108ee9;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Reply
            </div>
          </div>
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#ddd;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Cancel
            </div>
          </div>
        </div>
        <div
          class="am-swipe-actions am-swipe-actions-right"
        >
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#ddd;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Cancel
            </div>
          </div>
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#F4333C;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Delete
            </div>
          </div>
        </div>
        <div
          class="am-swipe-content"
        >
          <div
            class="am-list-item am-list-item-middle"
          >
            <div
              class="am-list-line"
            >
              <div
                class="am-list-content"
              >
                Have left and right buttons
              </div>
              <div
                class="am-list-extra"
              >
                More
              </div>
              <div
                aria-hidden="true"
                class="am-list-arrow am-list-arrow-horizontal"
              />
            </div>
            <div
              class="am-list-ripple"
              style="display:none;"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      style="background-color:gray;"
    >
      <div
        class="am-swipe"
      >
        <div
          class="am-swipe-cover"
        />
        <div
          class="am-swipe-actions am-swipe-actions-left"
        >
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#108ee9;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Reply
            </div>
          </div>
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#ddd;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Cancel
            </div>
          </div>
        </div>
        <div
          class="am-swipe-content"
        >
          <div
            class="am-list-item am-list-item-middle"
          >
            <div
              class="am-list-line"
            >
              <div
                class="am-list-content"
              >
                Only left buttons
              </div>
              <div
                class="am-list-extra"
              >
                More
              </div>
              <div
                aria-hidden="true"
                class="am-list-arrow am-list-arrow-horizontal"
              />
            </div>
            <div
              class="am-list-ripple"
              style="display:none;"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      style="background-color:gray;"
    >
      <div
        class="am-swipe"
      >
        <div
          class="am-swipe-cover"
        />
        <div
          class="am-swipe-actions am-swipe-actions-right"
        >
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#ddd;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Cancel
            </div>
          </div>
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#F4333C;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              Delete
            </div>
          </div>
        </div>
        <div
          class="am-swipe-content"
        >
          <div
            class="am-list-item am-list-item-middle"
          >
            <div
              class="am-list-line"
            >
              <div
                class="am-list-content"
              >
                Only right buttons
              </div>
              <div
                class="am-list-extra"
              >
                More
              </div>
              <div
                aria-hidden="true"
                class="am-list-arrow am-list-arrow-horizontal"
              />
            </div>
            <div
              class="am-list-ripple"
              style="display:none;"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      style="background-color:gray;"
    >
      <div
        class="am-swipe"
      >
        <div
          class="am-swipe-cover"
        />
        <div
          class="am-swipe-actions am-swipe-actions-right"
        >
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#ddd;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              short
            </div>
          </div>
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#F4333C;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              long text
            </div>
          </div>
        </div>
        <div
          class="am-swipe-content"
        >
          <div
            class="am-list-item am-list-item-middle"
          >
            <div
              class="am-list-line"
            >
              <div
                class="am-list-content"
              >
                Different button width
              </div>
              <div
                class="am-list-extra"
              >
                More
              </div>
              <div
                aria-hidden="true"
                class="am-list-arrow am-list-arrow-horizontal"
              />
            </div>
            <div
              class="am-list-ripple"
              style="display:none;"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      style="background-color:gray;"
    >
      <div
        class="am-swipe"
      >
        <div
          class="am-swipe-cover"
        />
        <div
          class="am-swipe-actions am-swipe-actions-right"
        >
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#ddd;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              button1
            </div>
          </div>
          <div
            class="am-swipe-btn "
            role="button"
            style="background-color:#F4333C;color:white;"
          >
            <div
              class="am-swipe-btn-text"
            >
              button2
            </div>
          </div>
        </div>
        <div
          class="am-swipe-content"
        >
          <div
            class="am-list-item am-list-item-middle"
          >
            <div
              class="am-list-line"
            >
              <div
                class="am-list-content"
              >
                List.Item with onClick
              </div>
              <div
                class="am-list-extra"
              >
                More
              </div>
              <div
                aria-hidden="true"
                class="am-list-arrow am-list-arrow-horizontal"
              />
            </div>
            <div
              class="am-list-ripple"
              style="display:none;"
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
